### DISCUSSION TITLE
Enable wayland in hangover-9.17

### DISCUSSION THREAD

**Discussion by fprietog** [AndreRH/hangover#165] [General]
Hello, 

@AndreRH

As we can see latest release hangover-9.17 came with this feature:
_"Wayland support in the packages"_

IMO users may think that just installing this package will make wine apps automatically runs on wayland instead of xwayland (when using wayland, of course) but, as I just tested, seems that it's necessary to create a registry key to make it works that way:

```
Windows Registry Editor Version 5.00
[HKEY_CURRENT_USER\Software\Wine\Drivers]
"Graphics"="wayland,x11"

```
The order of the drivers in the key determine the usage:
_"Graphics"="wayland,x11"_ make wayland be used by default:
![imagen](https://github.com/user-attachments/assets/9135cbe0-61cf-4f83-b3b8-fcb2dc01ca5b)

_"Graphics"="x11,wayland"_ make xwayland be used by default:
![imagen](https://github.com/user-attachments/assets/fcb6d1e8-facc-484a-9832-2678a0cfef84)

Notice that in wayland there are no decorations from the desktop theme but uses the wine selected ones.

I just only said this to advise that maybe a brief note in the README should be useful.

Thanks and best regards.

**Comment by AndreRH** [comment]
Hi, you're right. Sadly most people never read the readme, but I'll add that soon.

**Reply by AndreRH** [reply]
done

**Reply by fprietog** [reply]
Fantastic! Thank you very much!
